config/prod/JKG-s3.yaml

```diff
@@ -0,0 +1,39 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: JKG-Bucket
```

Comment:
We prefer this name to match the filename.  could you please either change this to `JKG-s3`? or change the filename to `JKG-Bucket.yaml`?

Reply:
```remote/JKG-Bucket.yaml```
or just
```JKG-Bucket.yaml```

Reply:
remote/JKG-Bucket.yaml